mantidimaging/gui/windows/stack_visualiser/presenter.py

```diff
@@ -106,7 +106,7 @@ def create_swapped_axis_stack(self):
             new_stack.name = self.images.name + "_sino"
             new_stack.record_operation(const.OPERATION_NAME_AXES_SWAP, display_name="Axes Swapped")
             self.view.parent_create_stack(new_stack)
-            new_dataset = StackDataset([new_stack], new_stack.name)
+            new_dataset = MixedDataset([new_stack], new_stack.name)
             self.view._main_window.presenter.model.add_dataset_to_model(new_dataset)
```

Comment:
Is something extra needed here now. Currently ( b0ff207bc ) if I load a dataset and do Image -> Create sinograms from stack. It makes the tab, and adds the stack to the model (I can see it in the operations window stack list), but it does not get an entry in the tree view.